e2e/tests/odoo-superset-flows.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { test, expect } from '@playwright/test';
+import { Odoo } from '../utils/functions/odoo';
+import { Superset } from '../utils/functions/superset';
+import { OpenMRS, patientName } from '../utils/functions/openmrs';
+import { O3_URL, SUPERSET_URL } from '../utils/configs/globalSetup';
+
+let odoo: Odoo;
+let openmrs: OpenMRS;
+let superset: Superset;
+
+test.beforeEach(async ({ page }) => {
+  openmrs = new OpenMRS(page);
+  odoo = new Odoo(page);
+  superset = new Superset(page);
+
+  await openmrs.login();
+  await openmrs.createPatient();
+  await openmrs.startPatientVisit();
+});
+
+test(`Creating an Odoo sale order line generates an entry in Superset's sale_order_lines table.`, async ({ page }) => {
+  // setup
+  await openmrs.searchPatient(`${patientName.firstName + ' ' + patientName.givenName}`);
+  await openmrs.goToLabOrderForm();
+  await page.getByPlaceholder('Search for a test type').fill('Complete blood count');
+  await openmrs.saveLabOrder();
```

Comment:
Do we have to involve the EMR/OpenMRS? Could you allow the tests manually create an Odoo customer, their quotations, add quotation lines e.t.c, there after verifying that those create entries in superset?

Reply:
Refer to this convo: https://mekomsolutions.slack.com/archives/G421UNF5L/p1726836155429709